### PR TITLE
Task lib version change for AzureFunctionApp, AzureFunctionAppContainer, AzureWebApp, AzureWebAppContainer

### PR DESCRIPTION
**Pull request by 20shivangi**

### CODE REVIEW

File: Tasks/AzureWebAppV1/make.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "rm": [
```
Comment (vincentdass) [inline Tasks/AzureWebAppV1/make.json]: remove rm

Comment (20shivangi) [reply Tasks/AzureWebAppV1/make.json]: It is done.

File: Tasks/AzureFunctionAppContainerV1/Tests/package-lock.json [hunk 2]
```diff
@@ -16,9 +16,9 @@
             "integrity": "sha1-TQJjewZ/6Vi9v906QOxW/vc3Mkc=",
             "dev": true,
             "requires": {
-                "assertion-error": "^1.0.1",
-                "deep-eql": "^0.1.3",
-                "type-detect": "^1.0.0"
+                "assertion-error": "1.0.2",
```
Comment (kmkumaran) [inline Tasks/AzureFunctionAppContainerV1/Tests/package-lock.json]: Should it remain 1.0.1?

File: Tasks/AzureFunctionAppContainerV1/package-lock.json [hunk 3]
```diff
@@ -20,124 +20,135 @@
       "integrity": "sha512-0WS7XU7sXzQ7J1nbnMKKYdjrrFoO3YtZYgUzeV8JFXffPnHfvSJQleR70I8BOAsOm14i4dyaAZ3YzqIl1YhkXQ=="
     },
     "abbrev": {
-      "version": "1.1.1",
-      "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
-      "integrity": "sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q=="
+      "version": "1.0.9",
```
Comment (kmkumaran) [inline Tasks/AzureFunctionAppContainerV1/package-lock.json]: Can we get rid of the package-lock changes?

File: Tasks/AzureFunctionAppV1/package-lock.json [hunk 4]
```diff
@@ -20,9 +20,9 @@
       "integrity": "sha512-0WS7XU7sXzQ7J1nbnMKKYdjrrFoO3YtZYgUzeV8JFXffPnHfvSJQleR70I8BOAsOm14i4dyaAZ3YzqIl1YhkXQ=="
     },
     "abbrev": {
-      "version": "1.1.1",
-      "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
-      "integrity": "sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q=="
+      "version": "1.0.9",
```
Comment (kmkumaran) [inline Tasks/AzureFunctionAppV1/package-lock.json]: Similar comment. Let's avoid these changes.

File: Tasks/AzureFunctionAppContainerV1/Tests/package-lock.json (hunk 2 above)
Comment (20shivangi) [reply Tasks/AzureFunctionAppContainerV1/Tests/package-lock.json]: Discussed offline

File: Tasks/AzureFunctionAppContainerV1/Tests/package-lock.json [hunk 5]
```diff
@@ -92,14 +92,14 @@
             "integrity": "sha1-EPIC5MwnYQLWDibjGfl9aNoJvpw=",
             "dev": true,
             "requires": {
-                "chai": ">=1.9.2 <4.0.0",
```
[the remark this answers is not in the document]
Comment (20shivangi) [reply Tasks/AzureFunctionAppContainerV1/Tests/package-lock.json]: Discussed offline

File: Tasks/AzureFunctionAppContainerV1/package-lock.json (hunk 3 above)
Comment (20shivangi) [reply Tasks/AzureFunctionAppContainerV1/package-lock.json]: Resolved offline

File: Tasks/AzureFunctionAppV1/package-lock.json (hunk 4 above)
Comment (20shivangi) [reply Tasks/AzureFunctionAppV1/package-lock.json]: Resolved offline

**Pull request merged**